README.md

```diff
@@ -376,6 +386,7 @@ Native desktop applications for managing and montoring docker hosts and clusters
 
 #### Web
 
+- [Admiral](https://vmware.github.io/admiral/) Web UI. By VMWare
```

Comment:
Please use the link and description of the repo here:

```suggestion
- [Admiral](https://github.com/vmware/admiral) Admiral™ is a highly scalable and very lightweight Container Management platform for deploying and managing container based applications. By [VMWare]
```

VMWare has to be moved to the links section on the bottom of the file due to duplication in Registry => Harbor